integreat_cms/cms/views/linkcheck/linkcheck.py

```diff
@@ -6,9 +8,10 @@
 from django.views.generic import ListView
 from django.views.generic.base import RedirectView
 
-from linkcheck.models import Link
+from linkcheck.models import Link, Url
 
 from ...utils.filter_links import filter_links
+from ...forms.linkchecker.edit_url_form import EditUrlForm
```

Comment:
```suggestion
from ...forms.linkcheck.edit_url_form import EditUrlForm
```
I'd rename the directory from `linkchecker` to `linkcheck` to be more consistent with the rest of our linkcheck files.